### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by jelly**
Test a few one line coverage cases, doesn't approve the coverage a lot. But it does test some important cases.

### CODE REVIEW

File: test/test_group.py [hunk 1]
```diff
@@ -207,6 +207,22 @@ def test_warn_on_add_group_with_package_already_having_open_group(db, client):
...
+    pkgnames = ['foo']
+    issues = ['CVE-1234-1234', 'CVE-2222-2222']
+    data = default_group_dict(dict(
+        cve='\n'.join(issues),
+        pkgnames='\n'.join(pkgnames),
+        fixed='0.8-1'))
+
+    resp = client.post(url_for('tracker.add_group'), follow_redirects=True, data=data)
+    assert 200 == resp.status_code
+    assert 'Version must be newer.' in resp.data.decode()
```
Comment (SantiagoTorres) [inline test/test_group.py]: I wonder if it'd make it easier to understand what's happening if we add an `affected` key to the data dictionary. Right now I don't know which is the affected version from the dictionary data or the decorators.

**Comment by anthraxx** [comment]
:heart: gonna review properly in the following days

File: test/test_group.py (hunk 1 above)
Comment (jelly) [reply test/test_group.py]: Done.

**Comment by anthraxx** [comment]
rebased and applied, thanks!

**Pull request closed without merging**